themes/origins/sections/slideshow.liquid

```diff
@@ -0,0 +1,279 @@
+{{ 'component-slider.css' | asset_url | stylesheet_tag }}
+{{ 'component-slider.js' | asset_url | script_tag_deferred }}
+
+{% assign slideshow = section.settings %}
+
+<yc-section
+  {% if slideshow.type == 'full' %}
+    data-full
+  {% endif %}
+  style="
+    --section-space-top: {{ slideshow.space_top }}px;
+    --section-space-bottom: {{ slideshow.space_bottom }}px;
+  "
+>
+  <yc-slider
+    {% if slideshow.autoplay %}
+      autoplay
+    {% endif %}
+    {% if slideshow.show_arrows %}
+      arrows
+    {% endif %}
+    {% if slideshow.type != 'full' %}
+      data-radius="true"
+    {% endif %}
+    interval="{{ slideshow.duration | times: 1000 }}"
+    data-type="full"
+    style="
+      --slider-gap: 0;
+      --slider-speed: 600;
+      --slider-per-page: 1;
+      --slider-height: {{ slideshow.height }};
+
+      --section-content-color: {{ slideshow.content_color.hex }};
+      --section-button-background: {{ slideshow.button_background.hex }};
+      --section-button-foreground: {{ slideshow.button_foreground.hex }};
+      --section-overlay-background: {{ slideshow.overlay_background.red }}, {{ slideshow.overlay_background.green }}, {{ slideshow.overlay_background.blue }};
```

Comment:
I was playing around with the overlay background, and tbh, it looks bad with any color other than black. Probably no need for this option, you know our sellers’ taste.

Reply:
Can't remember why we had it in the first place lol, classic theme doesn't have the option to change it. I Agree.